liquidity/liquidity.go

```diff
@@ -83,12 +83,9 @@ var (
 	// only be used for automatically dispatched swaps if autoloop is
 	// explicitly enabled, so we are happy to set a non-zero value here. The
 	// amount chosen simply uses the current defaults to provide budget for
-	// a single swap. We don't have a swap amount to calculate our maximum
-	// routing fee, so we use 0.16 BTC for now.
-	defaultBudget = defaultMaximumMinerFee +
-		ppmToSat(funding.MaxBtcFundingAmount, defaultSwapFeePPM) +
-		ppmToSat(defaultMaximumPrepay, defaultPrepayRoutingFeePPM) +
-		ppmToSat(funding.MaxBtcFundingAmount, defaultRoutingFeePPM)
+	// a single swap. We don't have a swap amount so we just use our max
+	// funding amount.
+	defaultBudget = ppmToSat(funding.MaxBtcFundingAmount, defaultFeePPM)
```

Comment:
Just out of curiosity: did the default budget increase or decrease with this change?

Reply:
Gone down a bit 💰